client/galaxy/scripts/components/Workflow/Invocations.vue

```diff
@@ -23,85 +21,63 @@
                 caption-top
                 :busy="loading"
             >
-                <template v-slot:table-caption>
-                    These invocations are not finished scheduling - one or more steps are waiting on others steps to be
-                    complete before the full structure of the jobs in the workflow can be determined.
-                </template>
                 <template v-slot:row-details="row">
                     <b-card>
-                        <!-- set provideContext to false, since the table itself provides this information -->
-                        <workflow-invocation-state :invocation-id="row.item.id" :provide-context="false" />
+                        <workflow-invocation-state :invocation-id="row.item.id" />
                     </b-card>
                 </template>
-                <template v-slot:cell(details)="data">
-                    <b-button
-                        v-b-tooltip.hover.bottom
-                        title="Show Invocation Details"
-                        class="btn-sm fa fa-chevron-down"
-                        v-if="!data.detailsShowing"
-                        @click.stop="swapRowDetails(data)"
-                    />
-                    <b-button
-                        v-b-tooltip.hover.bottom
-                        title="Hide Invocation Details"
-                        class="btn-sm fa fa-chevron-up"
-                        v-if="data.detailsShowing"
-                        @click.stop="swapRowDetails(data)"
-                    />
-                </template>
                 <template v-slot:cell(workflow_id)="data">
-                    <div v-if="!ownerGrid || !getWorkflowByInstanceId(data.item.workflow_id)">
-                        {{ data.item.workflow_id }}
-                    </div>
-                    <div v-else>
-                        <workflow-dropdown :workflow="getWorkflowByInstanceId(data.item.workflow_id)" />
-                    </div>
+                    <b-link href="#" @click.stop="swapRowDetails(data)">
+                        <b>{{ getWorkflowNameByInstanceId(data.item.workflow_id) }}</b>
+                    </b-link>
                 </template>
                 <template v-slot:cell(history_id)="data">
-                    <div v-if="!ownerGrid || !getHistoryById(data.item.history_id)">
-                        {{ data.item.history_id }}
-                    </div>
-                    <div v-else>
-                        <history-dropdown :history="getHistoryById(data.item.history_id)" />
-                    </div>
+                    {{ getHistoryNameById(data.item.history_id) }}
```

Comment:
At least have a way to navigate to the history?

Reply:
I added links for the history column which allows switching between histories.